### ISSUE TITLE
Unable to add url linkages from Author page

### ISSUE THREAD

**Issue by LeadSongDog** [internetarchive/openlibrary#1663]
### Please Describe the problem (or idea)
It is no longer possible to add VIAF or ISNI links such as https://viaf.org/viaf/111050448/ or http://isni.org/isni/0000000083470526 to the author record. 

> What happened just before the problem occurred? Or what problem could this idea solve?
The "Add Link" button showed the preview of the links, but they were not saved. Other information is saved.

> What did you expect to see that you didn't?
The links used to appear on the author page listed under the alternative names.

### Please show us where to look
https://openlibrary.org/authors/OL5456618A/Theophile_Stanley_Krawiec/edit

The link preview works:
![authlink](https://user-images.githubusercontent.com/11435431/49097690-45a16f00-f23b-11e8-87fb-bfd73a57469e.jpg)

The links are not added:
https://openlibrary.org/authors/OL5456618A/Theophile_Stanley_Krawiec?b=3&a=2&_compare=Compare&m=diff

This used to work:
https://openlibrary.org/authors/OL538507A/Robert_Simion_Feldman?m=diff&b=3

### Browser, version, and operating system
Chrome on Win7 desktop; also Chrome on iOS9

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

Possibly related to #1658

**Comment by jdlrobson** [comment]
Also broken on https://dev.openlibrary.org/books/OL21360361M/After_the_fact/edit ?

**Comment by jdlrobson** [comment]
I think the problem lies somewhere in openlibrary/plugins/openlibrary/js/jquery.repeat.js
It's using some hard to follow templating code

@tfmorris do you have any idea about how this code is supposed to work and what might be going wrong here?

**Comment by LeadSongDog** [comment]
Thank you. Now working as expected on production.
